packages/integration-tests/src/helpers/admin-tenant.ts

```diff
@@ -95,3 +107,29 @@ export const createUserAndSignInWithClient = async () => {
 
   return { id, client };
 };
+
+const createAndSignInToCloudClient = async (username: string, password: string) =>
```

Comment:
what does the create here stands for?

Reply:
this utils function should create a cloud client and sign in to it.

Reply:
Renamed as `createCloudClientAndSignIn`